hack/ginkgo-e2e.sh

```diff
@@ -100,6 +100,8 @@ fi
 # set before we get here, which eliminates any cluster/gke use if
 # KUBERNETES_CONFORMANCE_PROVIDER is set to "gke".
 if [[ -z "${NODE_INSTANCE_GROUP:-}" ]] && [[ "${KUBERNETES_PROVIDER}" == "gke" ]]; then
+  # TODO(pjh): this code is broken, detect-node-instance-groups appears nowhere
```

Comment:
........

Reply:
Should I file an issue about this and then I or someone can fix it later?

I'm pretty sure this code doesn't work... cc @mtaufen

Reply:
You can file an issue or send a separate PR later to remove the code.